src/summary/documentation/dimensions-card-level-two.component.tsx

```diff
@@ -3,16 +3,25 @@ import styles from "./dimensions-card-level-two.css";
 import SummaryCard from "../cards/summary-card.component";
 import { getDimensions } from "./dimensions-card.resource";
 import { match } from "react-router";
+import { useCurrentPatient } from "@openmrs/esm-api";
 
 function DimensionsCardLevelTwo(props: DimensionsCardLevelTwoProps) {
   const [dimensions, setDimensions] = React.useState([]);
+  const [
+    isLoadingPatient,
+    patient,
+    patientUuid,
+    patientErr
+  ] = useCurrentPatient();
```

Comment:
Getting patientUuid from hook instead of from props means that we can render this component **before** we have the patient object. This is called the [Fetch as you render approach](https://reactjs.org/docs/concurrent-mode-suspense.html#approach-1-fetch-on-render-not-using-suspense) to data fetching in react. The downside of the approach is what the docs call a "cascade effect" where you don't render the child components until the top level data is available. Switching to `useCurrentPatient` makes it more possible for us to avoid the "cascade effect" that has the negative effect. Note that it doesn't totally solve the cascade problem - it just makes it easier for us to avoid it where possible. In the case of dimensions and allergies and stuff, we unfortunately have to first load the patient before we even have the identifier needed to load the other stuff.

This pattern I'm doing here has been transferred over to a bunch of other components, too